ui/app/AppLayouts/Profile/views/PrivacyAndSecurityView.qml

```diff
@@ -39,7 +33,7 @@ SettingsContentBase {
                     id: enableMetricsSwitch
                     checked: root.isCentralizedMetricsEnabled
                     onClicked: {
-                        Global.openMetricsEnablePopupRequested(Constants.metricsEnablePlacement.privacyAndSecurity, popup => popup.toggleMetrics.connect(refreshSwitch))
+                        Global.openMetricsEnablePopupRequested(Constants.metricsEnablePlacement.privacyAndSecurity, popup => popup.closed.connect(refreshSwitch))
```

Comment:
This actually fixes a bug: if you clicked on the switch directly (causing it to flip to enabled) and then closed the popup with Esc key, the Switch would stay on!

Reply:
There is a workaround used that unnecessarily complicates code and can be easily fixed. The key point is to use `onToggled` instead of `onClicked`, then `refreshSwitch` nor callback are not needed.

I attach simple test showing difference between `onClicked` (breaking binding) and `onToggled` (preserving binding):

```qml
    property bool selected: true

    Switch {
        checked: root.selected

        //onClicked: root.selected = !root.selected
        onToggled: root.selected = !root.selected
    }

    Switch {
        y: 40

        enabled: false
        checked: root.selected
    }
```

On clicked:

[clicked.webm](https://github.com/user-attachments/assets/03f39623-3ad3-46bd-af24-02f513e67f4a)

On toggled:

[toggled.webm](https://github.com/user-attachments/assets/0f91a521-d0a7-48d2-882e-e56e06252b70)